src/app/plugins/platform/JavaScript/templates/tasks/webpack.js

```diff
@@ -6,6 +6,6 @@ module.exports = async function webpackBuild() {
   const webpackConfig = require('../webpack.config')({mode: 'production'});
   const stats = await asyncWebpack(webpackConfig);
   if (stats.hasErrors()) {
-    throw new Error(stats.compilation.errors.join('\n'));
+    throw new Error(stats.compilation.errors.map(i => i.rawMessage).join('\n'));
```

Comment:
Let's call this parameter `err` instead of `i` to better denote what we're mapping over.

Reply:
Thanks for catching this. Still trying to unlearn this habit.